capstone-backend/src/main/java/com/google/sps/servlets/ClubServlet.java

```diff
@@ -12,11 +12,19 @@
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
 import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
```

Comment:
The imports 

```
import com.google.appengine.api.datastore.Query.SortDirection;
import com.google.appengine.api.users.UserService;
import com.google.appengine.api.users.UserServiceFactory;

import com.google.gson.GsonBuilder;
import com.google.gson.JsonArray;

import java.util.Enumeration;
import java.util.Arrays;
```
Are never utilized and can be removed.

Reply:
Good catch thank you